autogpt/config/config.py

```diff
@@ -205,6 +207,19 @@ def build_config_from_env(cls):
                 "disabled_command_categories"
             ] = disabled_command_categories.split(",")
 
+        wipe_workspace_on_start = os.getenv("WIPE_WORKSPACE_ON_START")
+        if wipe_workspace_on_start == "True":
```

Comment:
Probably better to update config_dict here, and leave move the actual workspace path clearance to the Workspace module.
For a couple of reasons:
1. `workspace_directory` is a cli option (hidden for now, but likely to be exposed soon)
2. `build_config_from_env` shouldn't have side effects apart from initializing the config class

Reply:
Agreed, I propose adding a `wipe` parameter to `Workspace.__init__` and doing it in there.